src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutConnectorCheck.java

```diff
@@ -0,0 +1,156 @@
+package org.openstreetmap.atlas.checks.validation.linear.edges;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.Heading;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.Edge;
+import org.openstreetmap.atlas.geography.atlas.walker.OsmWayWalker;
+import org.openstreetmap.atlas.tags.HighwayTag;
+import org.openstreetmap.atlas.tags.JunctionTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+import org.openstreetmap.atlas.utilities.scalars.Angle;
+
+/**
+ * This check looks for roundabout connectors that intersect the roundabout at too sharp an angle.
+ * This is generally caused by one way roads drawn in the wrong direction or two way roads that
+ * should be one way.
+ *
+ * @author bbreithaupt
+ */
+public class RoundaboutConnectorCheck extends BaseCheck<Long>
+{
+    private static final long serialVersionUID = -5311314357995383430L;
+
+    private static final Double ONE_WAY_THRESHOLD_DEFAULT = 100.0;
+    private static final Double TWO_WAY_THRESHOLD_DEFAULT = 130.0;
+    private static final String ONE_WAY_INSTRUCTION = "This way, id:{0,number,#}, is connected to a roundabout at too sharp and angle. It may be digitized backwards";
```

Comment:
typo: is connected to a roundabout at too sharp ~and~ an angle. 
Same for the below instruction.